tool/evaluate-server.js

```diff
@@ -162,7 +162,12 @@ module.exports = {
             required: false,
             type: Number,
             default: 0,
-            help: 'Collapse all examples of complexity smaller or equal to this'
+            help: 'Collapse all examples of complexity smaller or equal to this',
+        });
+        parser.addArgument('--oracle', {
```

Comment:
We recently updated argparse, the new API is called `add_argument` (it will show a deprecation warning otherwise)